data/tutorials/en/introOCaml.md

```diff
@@ -0,0 +1,16 @@
+# Introduction to OCaml
+
+OCaml is a versatile and efficient functional programming language that's increasing in popularity across the functional programming community. It just celebrated its 25th anniversary in 2021, as Xavier Leroy released OCaml on May 9, 1996 in collaboration with Jérôme Vouillon, Damien Doligez, and Didier Rémy, and it has steadily grown over the past few decades. Thus far, it has been mostly taught in computer science programs at universities, but we'd like to make it more accessible to the greater programming community. The following documentation gives an introduction to OCaml, teaches the basics, explains the similarities and differences to the more popular imperative languages, and outlines OCaml's libraries and syntax. You'll soon find OCaml is a straightforward, simple language to learn. 
```

Comment:
> that's increasing in popularity across the functional programming community

Although I don't have precise metrics, I think it's it's growing in popularity across the general programming community 🙂

Reply:
> Thus far, it has been mostly taught in computer science programs at universities, but we'd like to make it more accessible to the greater programming community.

This is not a very good selling point for an intro 😁  But also, I don't think that's true, there's a lot of industrial usage of OCaml, as shown in the [industrial users page](https://v3.ocaml.org/industrial-users).